site/static/main.js

```diff
@@ -1,36 +1,34 @@
 /* tslint:disable:no-unused-variable */
```

Comment:
Because we can check that the code is compatible. Mainly, I like that we don't have any js in the repo. Makes it easier.